dpgen2/superop/prep_run_fp.py

```diff
@@ -125,6 +125,13 @@ def _prep_run_fp(
     run_template_config = run_config.pop("template_config")
     prep_executor = init_executor(prep_config.pop("executor"))
     run_executor = init_executor(run_config.pop("executor"))
+    template_slice_config = run_config.pop("template_slice_config", None)
+    group_size = (
+        template_slice_config.get("group_size", None) if template_slice_config else None
+    )
+    pool_size = (
+        template_slice_config.get("pool_size", None) if template_slice_config else None
+    )
```

Comment:
The two keys are not necessary when the `template_slice_config` is correctly implemented.